GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -326,9 +325,51 @@ private static Process StartProcessAndReadAllBytes(string arguments, string cmd,
             return process;
         }
 
+        private static async Task<(Process, byte[] stdOut, byte[] stdErr)> StartProcessAndReadAllBytesAsync(string arguments, string cmd, string workDir, byte[] stdInput)
+        {
+            if (string.IsNullOrEmpty(cmd))
+                return (null, null, null);
```

Comment:
:bulb: Use named values

Reply:
Matter of taste. Do you prefix all your method arguments with explicit parameter names? To me this is the same.